catalog/opensearch/catalog-opensearch-source/src/main/java/org/codice/ddf/opensearch/source/OpenSearchSource.java

```diff
@@ -861,7 +861,7 @@ public void setDistanceTolerance(int distanceTolerance) {
   }
 
   public void setPollInterval(Integer interval) {
-    this.pollInterval = interval;
+    this.pollInterval = Math.max(1, interval);
```

Comment:
✏️ Not sure if you agree, but should we have a log statement here that says it's defaulting to 1 if the interval is less than 1?